### PR TITLE
Give out error message when meet errors

### PR DESCRIPTION
**Pull request by js0701**
It will help for debug HWC.
Aslo fix android build warning cutils/log.h is deprecated

Change-Id: I2937a0bb15e43da3d05557a5aa0efb574635ac09
Jira:None
Tests:None
Signed-off-by: Lin Johnson <johnson.lin@intel.com>

### CODE REVIEW

File: os/android/utils_android.h [hunk 1]
```diff
@@ -180,20 +180,9 @@ static uint32_t DrmFormatToHALFormat(int format) {
 }
 
 static native_handle_t *dup_buffer_handle(buffer_handle_t handle) {
```
Comment (kalyankondapally) [inline os/android/utils_android.h]: Can u remove Change-Id: Ic5ea88e7271c7a58e6afaf59f798e74bc41a33f6
Tests: Build passes on Android  
?

File: common/compositor/va/vautils.cpp [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
 #include <drm_fourcc.h>
 
+#include <hwctrace.h>
+
```
Comment (kalyankondapally) [inline common/compositor/va/vautils.cpp]: Change-Id: I2937a0bb15e43da3d05557a5aa0efb574635ac09
Tests: Build passes on Android  
?

Comment (js0701) [reply common/compositor/va/vautils.cpp]: done

File: os/android/utils_android.h (hunk 1 above)
Comment (js0701) [reply os/android/utils_android.h]: done

**Comment by kalyankondapally** [comment]
merged

**Pull request closed without merging**